src/context/theme/ThemeController.tsx

```diff
@@ -0,0 +1,55 @@
+import React, {Fragment, PropsWithChildren, useEffect} from 'react';
+
+import blockOrigin from 'bem-cn-lite';
+
+import {Theme} from '../../models';
+
+import {useTheme} from './useTheme';
+
+const b = blockOrigin('yc-root');
+
+export interface CommonThemeControllerProps {
+    children?: React.ReactNode;
+}
+
+export const GlobalThemeController = ({children}: CommonThemeControllerProps) => {
```

Comment:
it is used in storybook only, could we move it to storybook folder?

Reply:
Of course